Formula/frps.rb

```diff
@@ -0,0 +1,72 @@
+class Frps < Formula
+  desc "This is the server of frp"
+  homepage "https://github.com/fatedier/frp"
+  url "https://github.com/fatedier/frp.git",
+      :tag      => "v0.24.1",
+      :revision => "cbf9c731a096ce33ec282ed913a25cb2e5f05303"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["XC_ARCH"] = "amd64"
```

Comment:
Is this needed to build succesfully?

Reply:
@xykong please take a look

Reply:
@qiwihui thanks for remind, just fix it.